src/servant/servant_unk0.h

```diff
@@ -0,0 +1,21 @@
+// SPDX-License-Identifier: AGPL-3.0-or-later
+s32 ServantUnk0(void) {
```

Comment:
Had no idea what to call this, seems like it's some sort of check against an enum value most likely. Calling code didn't shed much light.